src/CampaignAdmin.php

```diff
@@ -156,6 +158,37 @@ public function readCampaigns()
     }
 
     /**
+     * @return array
+     */
+    protected function getPlaceholderGroups()
+    {
+        $groups = [];
+
+        $classes = Config::inst()->get(ChangeSet::class, 'important_classes');
+        
+        foreach ($classes as $class) {
+            /** @var DataObject $item */
+            $item = Injector::inst()->get($class);
```

Comment:
get() won't return null if the class doesn't exist, it'll throw an exception. Can you use class_exists() instead please?